spras/analysis/summary.py

```diff
@@ -71,7 +108,7 @@ def summarize_networks(file_paths: Iterable[Path], node_table: pd.DataFrame, alg
         nw_info.append(cur_nw_info)
 
     # Prepare column names
-    col_names = ['Name', 'Number of nodes', 'Number of edges', 'Number of connected components']
+    col_names = ['Name', 'Number of nodes', 'Number of edges', 'Number of connected components', 'Density', 'Max degree', 'Median degree', 'Max diameter', 'Average path length']
```

Comment:
As we add more summary statistics, we might want to use some kind of more modular system for adding statistics: the current `cur_mw_info` aligned with `col_names` approach looks pretty bad.

Reply:
Would you say that is out of scope for this PR and should be a new issue?